client-app/app-runner.ts

```diff
@@ -59,8 +59,10 @@ export default async (getPlugins: (options: any) => { plugin: Plugin; options: a
   app.use(contextPlugin, themeContext.value);
   app.use(configPlugin, themeContext.value!.settings);
 
-  const plugins = getPlugins({ router });
-  plugins.forEach(({ plugin, options }) => app.use(plugin, options));
+  if (getPlugins) {
```

Comment:
No need to check, default empty function is defined in line 19

Reply:
i have fixed it